Mage.Tests/src/test/java/org/mage/test/cards/abilities/oneshot/destroy/UnlicensedDisintegrationTest.java

```diff
@@ -0,0 +1,115 @@
+package org.mage.test.cards.abilities.oneshot.destroy;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+
+/*
+Unlicensed Disintigration - Hi! Noticed that everytime that i succesfully cast 
+Unlicensed Disintigration with an artifact on the board the opponent wont lose 3 life. 
+The creature dies but the last piece of text does not work (teM, 2020-02-24 15:17:36)
+*/
+public class UnlicensedDisintegrationTest extends CardTestPlayerBase{
+    
+    @Test
+    public void testDestroyCreatureLifeLoss(){
+        addCard(Zone.BATTLEFIELD, playerA, "Mountain");
+        addCard(Zone.BATTLEFIELD, playerA, "Swamp");
+        addCard(Zone.BATTLEFIELD, playerA, "Swamp");
+        addCard(Zone.HAND, playerA, "Unlicensed Disintegration");
```

Comment:
Add testing/using card's abilities text in the tests as comments. If you cast or activate it then add cost too. Example:
![shot_200607_191239](https://user-images.githubusercontent.com/8344157/83972725-e07c0e80-a8f2-11ea-8057-b2ae90b3c123.png)

If you use same card in many tests then you can add card's description as a class comment. Example:
![shot_200607_191040](https://user-images.githubusercontent.com/8344157/83972694-b0347000-a8f2-11ea-886d-ee261db0093b.png)

Also I recommends to change add card code order to "card -> lands for pay -> targets". It's easy to read test code. See above example.